pipelines/templates/publishpackages.yml

```diff
@@ -1,17 +1,23 @@
-# [Template] Publish packages to artifacts and to NuGet feed
+# [Template] Publish packages to artifacts, UPM registry and NuGet feed
 
 steps:
 - task: PublishBuildArtifacts@1
   displayName: 'Publish Packages'
   inputs:
     PathtoPublish: '$(Build.SourcesDirectory)\artifacts'

```

Comment:
Should the UPM packages be published to `artifacts` here too?

Reply:
i dont think so. the unitypackage files look to be pushed twice.